src/tests/integration/IntegrationBase.t.sol

```diff
@@ -386,9 +386,9 @@ abstract contract IntegrationBaseTest is DssTest {
         // Rest of the liquidity can be withdrawn
         hub.exec(ilk);
         vow.heal(_min(vat.sin(address(vow)), vat.dai(address(vow))));
-        assertEq(vat.gem(ilk, address(end)), 0);
+        assertRoundingEq(vat.gem(ilk, address(end)), 0);
         assertEq(vat.sin(address(vow)), 0);
-        assertGe(vat.dai(address(vow)), prevDai); // As also probably accrues interest
+        //assertGe(vat.dai(address(vow)), prevDai); // As also probably accrues interest
```

Comment:
same question as below?

Reply:
Same. Rounding error.

Reply:
We can replace the whole line by this one (removing the comment as I'm not sure why it mentions that probably accrues interest when the time is static):
```
assertApproxEqAbs(vat.dai(address(vow)), prevDai, roundingTolerance * RAY);
```

Reply:
re-enabled, but relaxed the assert

Reply:
LGTM